### PR TITLE
Documentation: add tutorial to set up a static IP address

### PR DESCRIPTION
**Pull request by gvancuts**
Add a tutorial on how to change the default configuration that
uses DHCP and assign a static IP address to the system.

Signed-off-by: Geoffroy Van Cutsem <geoffroy.vancutsem@intel.com>

### CODE REVIEW

**Comment by NanlinXie** [comment]
Can one of the admins verify this patch?

**Comment by gvancuts** [comment]
@bem-nrl could you take a quick look at this (since you're the first one that used this :-) and let me know if you have any comment. Thanks in advance!

**Comment by gvancuts** [comment]
Thanks for the quick review @bem-nrl !

File: doc/tutorials/static-ip.rst [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+bridge called ``acrn-br0`` will be created and attached to the Ethernet network
+interface of the platform. By default, the bridge gets its network configuration
+using DHCP. This guide will explain how to modify the system to use a static IP
+address. You need ``root`` privileges to make these changes to the system.
+
+ACRN Network Setup
+******************
+
+The ACRN Service OS is based on `Clear Linux`_ and it uses `systemd-networkd`_
+to set-up the Service OS networking. A few files are responsible for setting up the
```
Comment (dbkinder) [inline doc/tutorials/static-ip.rst]: ``set up`` (two words)

File: doc/tutorials/static-ip.rst [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+ACRN bridge (``acrn-br0``), the TAP device (``acrn_tap0``) and how these are all
```
Comment (dbkinder) [inline doc/tutorials/static-ip.rst]: add a comma before the and:

    ... (``acrn_tap0``), and how ...

File: doc/tutorials/static-ip.rst [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+connected. Those files are installed in ``/usr/lib/systemd/network``
+on the target device and can also be found under ``tools/acrnbridge`` in the source code.
+
+Setting up the static IP address
+********************************
+
+In order to set-up a static IP address, you need to modify the
```
Comment (dbkinder) [inline doc/tutorials/static-ip.rst]: Simplify the wording a bit with:

    You can set up a static IP address by modifying the ...

File: doc/tutorials/static-ip.rst [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+   :caption: tools/acrnbridge/eth.network
+   :emphasize-lines: 5
+
+Edit the file to remove the line highlighted above and add your network settings in
+that ``[Network]`` section. You will typically need to add the ``Address=``, ``Gateway=``
+and ``DNS=`` parameters in there. There are many more parameters that can be used and
+detailing them is beyond the scope of this document. For an extensive list of those,
+please visit the official `systemd-network`_ page.
+
+This is an example of what a typical ``[Network]`` section would look like:
```
Comment (dbkinder) [inline doc/tutorials/static-ip.rst]: for clarity add a bit at the end:

    ... would look like, specifying a static IP address:

File: doc/tutorials/static-ip.rst [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+   [Network]
+   Address=192.168.1.87/24
+   Gateway=192.168.1.254
+   DNS=192.168.1.254
+
+Activate the new configuration
+******************************
+
+You do not need to reboot the machine after making the changes to the system, the
+following steps will suffice (run as ``root``).
```
Comment (dbkinder) [inline doc/tutorials/static-ip.rst]: clarify with:

    following steps that restart the systemd-networkd daemon will suffice (run as ``root``):

File: doc/tutorials/static-ip.rst [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+.. code-block:: console
+
+   systemctl daemon-reload
+   systemctl restart systemd-networkd
+
+If you encounter connectivity issues after following this guide, please contact us on the
+`ACRN-users mailing list`_ or file an issue in `ACRN hypervisor issues`_. Provide the details
+of the configuration you are trying to set up, the modifications you have made to your system and
```
Comment (dbkinder) [inline doc/tutorials/static-ip.rst]: add a comma before and:

     ... made to your system, and

**Comment by gvancuts** [comment]
@dbkinder just force-pushed an update that addresses all your comments. PTAL. Thanks!
